erpnext/stock/doctype/shipment/api/utils.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
```

Comment:
This entire utils.py file can be kept outside along with shipment.js, shipemnt.py, etc.
Conventionally we do not have a `api` folder inside any doctype
So, just like doctype `pricing_rule` has utils in it, this should also just be kept inside the shipment folder.